can/message.py

```diff
@@ -279,8 +279,21 @@ def _check(self):
                     "error state indicator is only allowed for CAN FD frames"
                 )
 
+    def __eq__(self, other: Any) -> bool:
+        """ Compares a given message with this one.
+        The result is the as same as calling ``equals`` without timestamp and channel check.
+
+        :param other: the message to compare with
+
+        :return: True if the given message equals this one
+        """
+        return self.equals(other, timestamp_delta=None, check_channel=False)
```

Comment:
No additional arguments should be passed here so the behavior is always the default.

Reply:
Ok. I was not shure if checking the timestamp should be done or not. So I'll remove the both arguments, and make check_channel True. Then this is a full-property compare (with the recommended float-compare with an epsilon value)

Reply:
There was a discussion earlier and the conclusion was to compare everything by default as that results in less surprises. In a real world application you might however want to relax some checks so that’s why the check method exists.

Reply:
Yup. Please have a look on the last commit.